instruqt/replicated-cli/02-verifying-manifests/assignment.md

```diff
@@ -81,8 +81,8 @@ to have as available resources in the cluster. We have a pretty
 simple application, so we won't ask for much.
 
 Most times, you'll do this interactively in your favorite editor.
-To simplify for the lab, we're going to copy in a file where
-we've already made the changes.
+To simplify for the lab, we're going use some command-line fu to
```

Comment:
what's 'fu'?

Reply:
Kind of old way of saying you're going to exercise your command line skills. Like kung fu is a fighting skill that can seem like magic in the movies, command-line fu is a technical skill that can seem like magic

Reply:
Changing to something a little less esoteric